src/sentry/issues/group_policy.py

```diff
@@ -0,0 +1,167 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import TYPE_CHECKING, Any, Dict, Optional, Set, Type
+
+from django.conf import settings
+
+from sentry.issues.grouptype import GroupType
+from sentry.models import Organization, Project
+from sentry.utils import metrics, redis
+
+if TYPE_CHECKING:
+    from sentry.utils.performance_issues.performance_detection import PerformanceProblem
+
+_group_policy_registry: Dict[int, Type[GroupPolicy]] = {}
+DEFAULT_IGNORE_LIMIT: int = 3
+DEFAULT_EXPIRY_TIME: int = 60 * 60 * 24
+
+
+@dataclass(frozen=True)
+class NoiseConfig:
+    ignore_limit: int = DEFAULT_IGNORE_LIMIT
+    expiry_time: int = DEFAULT_EXPIRY_TIME
+
+
+@dataclass(frozen=True)
+class GroupPolicy:
+    group_type_id: int
+    organizations: Optional[Dict[str, NoiseConfig]] = None
```

Comment:
Is the str here the organization slug? It might be more stable to use the org id instead here, since the slug is mutable.

Reply:
ah hm yeah. I just wanted to make it more human readable

Reply:
looks like the EA/LA list is slugs as well so I might revert back to the slugs, but pull from getsentry as appropriate